examples/js/loaders/MMDLoader.js

```diff
@@ -67,6 +68,23 @@ THREE.MMDLoader.prototype.defaultToonTextures = [
 	'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAL0lEQVRYR+3QQREAAAzCsOFfNJPBJ1XQS9r2hsUAAQIECBAgQIAAAQIECBAgsBZ4MUx/ofm2I/kAAAAASUVORK5CYII='
 ];
 
+/*
+ * getImageData() of CanvasRenderingContext2D with image in other domain
+ * for transparency detection fails because of cross-origin problem
+ * even if server responds with "Access-Control-Allow-Origin: *".
+ * If this.imageCrossOrigin is true, MMDLoader tries to solve this issue
+ * with
+ *   var image = new Image();
+ *   image.crossOrigin = 'anonymous';
+ *   image.src = url;
+ * This workaround can be removed if ImageLoader supports crossOrigin.
```

Comment:
`ImageLoader.crossOrigin` isn't used for anything now

https://github.com/mrdoob/three.js/blob/c2cec2e7dca8b6b47ce25f831635b13275e0217f/src/loaders/ImageLoader.js#L58

Even if I set `image.crossOrigin = 'anonymous';` right before `image.src = URL.createObjectURL( blob );`, we can't avoid cross-origin error on `getImageData()` of `CanvasRenderingContext2D`.

https://github.com/mrdoob/three.js/blob/c2cec2e7dca8b6b47ce25f831635b13275e0217f/src/loaders/ImageLoader.js#L46

It seems like `image.crossOrigin = 'anonymous';` won't work for ``image.src = URL.createObjectURL( blob );`. I speculate cross-origin info is decided at the `URL.createObjectURL` or `XMLHttpRequest.send` moment and the info will be just propagated to `image`. Probably `image.crossOrigin = 'anonymous';` can't block this propagation.